### PR TITLE
Update VSCode extension docs

### PR DESCRIPTION
**Pull request by gj**

### CODE REVIEW

File: docs/content/any/reference/tooling/ide.md [hunk 1]
```diff
@@ -2,18 +2,33 @@
...
 ### [Vim](https://github.com/osohq/polar.vim)
 
+#### Features
+
+- Syntax highlighting.
+
 ### Want support for your IDE of choice?
 
-Let us know by using our chat on the bottom right to send us a Slack message.
+<!-- TODO(gj): is this still accurate? I don't see the Intercom button on this page. -->
```
Comment (gj) [inline docs/content/any/reference/tooling/ide.md:33]: Highlighting for 👀

Comment (dhatch) [reply docs/content/any/reference/tooling/ide.md:33]: Yeah your browser is probably just blocking it.

Comment (dhatch) [reply docs/content/any/reference/tooling/ide.md:33]:
```suggestion
Let us know by using our chat on the bottom right to send us a Slack message.
```

File: docs/spelling/allowed_words.txt [hunk 2]
```diff
@@ -85,6 +85,7 @@ URI
 URIs
 VM
 VPN
+VSCode's
```
Comment (dhatch) [inline docs/spelling/allowed_words.txt:88]: is `'s` needed here? 
```suggestion
VSCode
```

File: vscode/oso/README.md [hunk 3]
```diff
@@ -6,7 +6,10 @@ Syntax highlighting for `.polar` files as used by [Oso](https://www.osohq.com).
 
 ## Language Server Protocol (LSP) Functionality
```
Comment (dhatch) [inline vscode/oso/README.md:7]: Not sure that users care that this is using LSP protocol.

**Review by dhatch — APPROVED** [review]
LGTM mostly - a few small changes to make before merging.

File: docs/spelling/allowed_words.txt (hunk 2 above)
Comment (gj) [reply docs/spelling/allowed_words.txt:88]: yes lol — blame whatever spelling lib we're using

File: vscode/oso/README.md (hunk 3 above)
Comment (gj) [reply vscode/oso/README.md]: 1369607a

**Review by dhatch — APPROVED** [review]

**Pull request merged**